release/ray_release/buildkite/filter.py

```diff
@@ -36,8 +37,13 @@ def filter_tests(
                 break
         if attr_mismatch:
             continue
-        if not run_jailed_tests and test.get("jailed", False):
-            continue
+        if not run_jailed_tests:
+            if test.get("jailed", False):
+                continue
+            test.update_from_s3()
+            sm = TestStateMachine(test)
+            if test.is_jailed_with_open_issue(sm.ray_repo):
```

Comment:
why `ray_repo` comes from `sm`? I would expect it is either a constant or comes from the `test`?

Reply:
I'll move it to a class method so that it's less awkward